packages/manager/apps/account-disable-2fa/src/pages/create/form/FormDocumentFields/FormDocumentFieldList.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { FunctionComponent } from 'react';
+import {
+  Control,
+  Controller,
+  FieldValues,
+  UseFormSetError,
+} from 'react-hook-form';
+import { FormDocumentFieldItem } from './FormDocumentFieldItem';
+import documentsFieldRules from './documentFieldsConfiguration.json';
+import { FileWithError } from '@/components/FileInput/FileInputContainer';
+
+type Props = {
+  legalForm: string;
+  subsidiary: string;
+  control: Control<FieldValues>;
+};
+
+type DocumentsFieldRules = {
+  [key: string]: {
+    [key: string]: {
+      fields: {
+        field: string;
+        multiple: boolean;
+        tooltipCount: number;
+      }[];
+    };
+  };
+};
```

Comment:
```suggestion
type LegalFrom = 'individual' | 'association' | 'corporation' | 'other' | 'administration';
type Sub = 'FR' | 'DE' | 'ES' | 'PL' | 'IE' | 'IT' | 'NL' | 'PT' | 'GB' | 'CZ' | 'FI' | 'LT' | 'LTE' | 'CA' | 'QC' | 'WS' | 'TN' | 'MA' | 'SN' | 'AU' | 'ASIA' | 'SG' | 'IN' | 'WE';

type Props = {
  legalForm: LegalFrom | 'default';
  subsidiary: Sub | 'DEFAULT';
  control: Control<FieldValues>;
};

type FieldDefinition = {
  field: string;
  multiple: boolean;
  tooltipCount: number;
};

type FieldsListDefinition = {
  fields: FieldDefinition [];
};

type DocumentsFieldRules = Record<LegalFrom | 'default', Record<Sub | 'DEFAULT', FieldsListDefinition >>;
```

@anooparveti do you agree with the usage of this kind of type definition for `LegalForm` and `Sub` ?
If we I think this proposition would be clearer